app/controllers/idv/sessions_controller.rb

```diff
@@ -31,7 +44,7 @@ def start_idv_session
     end
 
     def applicant_from_params
-      app_vars = applicant_params.delete_if { |_key, value| value.blank? }
+      app_vars = idv_params.merge(financial_params.delete_if { |_key, value| value.blank? })
```

Comment:
the single form is being split into 2 separate forms, 2 separate actions, etc. So the basic applicant params and financial params are split for input, then merged together to instantiate the `Proofer::Applicant` on the 2nd submit.